eras/babbage/impl/testlib/Test/Cardano/Ledger/Babbage/Imp/UtxoRefScriptSpec.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Test.Cardano.Ledger.Babbage.Imp.UtxoRefScriptSpec (
```

Comment:
No I really meant that we should have one spec per rule.
```suggestion
module Test.Cardano.Ledger.Babbage.Imp.UtxoSpec (
```

Reply:
If the module does grow too big at some point we'll split it up into submodules, until then let's organize all of the Imp tests on the per rule basis